### PR TITLE
Windu

### PR DESCRIPTION
**Pull request by 2ndborn**
Hello There Gif

# Pull Request

❗ I have added the problem statement to the readme doc❗️

## Pull Request details:

- Closes issue: #

- INFO ABOUT THE WORK IN YOUR PULL REQUEST GOES HERE (Please be as descriptive as possible)

## Any Breaking changes:

- IF ANYTHING YOU'RE COMMITTING WOULD BREAK SOMETHING, INCLUDE HERE WHAT WOULD BREAK
- IF YOU HAVE NO BREAKING CHANGES, ENTER 'None'

## Associated Screenshots:

_( Feel free to add gifs/png screenshots of your feature in action 😊 )_

- IF YOU HAVE ANY SCREENSHOTS, INCLUDE THEM HERE.
- IF YOU HAVE NO SCREENSHOTS, ENTER 'None'

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -108,6 +108,33 @@ footer a:hover {
   background-size: cover;
 }
 
+<<<<<<< HEAD
```
Comment (kera-cudmore) [inline assets/css/style.css:111]: Can you remove this please 😊

File: assets/css/style.css [hunk 2]
```diff
@@ -108,6 +108,33 @@ footer a:hover {
...
+
+#wiki-text {
+  font-family: "rubik", sans-serif;
+  font-size: 25px;
+  color: var(--red);
+  text-shadow: 3px 3px var(--black);
+  visibility: hidden;
+}
+  
+=======
```
Comment (kera-cudmore) [inline assets/css/style.css:136]: And here to line 137

**Review by kera-cudmore — CHANGES_REQUESTED** [review]

File: assets/css/style.css (hunk 1 above)
Comment (2ndborn) [reply assets/css/style.css:111]: Done

File: assets/css/style.css (hunk 2 above)
Comment (2ndborn) [reply assets/css/style.css:136]: Done

**Review by kera-cudmore — APPROVED** [review]

**Pull request merged**